content/bots/twitterbots/_sys_bot_.md

```diff
@@ -0,0 +1,16 @@
+/*
+Title: @_sys_bot_
+Description: Potential LCD Soundsystem tribute band names
+Author: botsheeter.py
+Date: October 26, 2015
+Tags: twitter,twitterbot,active,python,rhyming,rhyme,band_names,open source,opensource,python,swizzard
+Nav: hidden
+Robots: index,follow
+*/
+
+[![](/content/bots/twitterbots/images/_sys_bot_.png)](https://twitter.com/_sys_bot_)
+
+[@_sys_bot_](https://twitter.com/_sys_bot_) is an [open source](https://github.com/swizzard/sys_bot) Twitter bot created by [swizzard](https://twitter.com/http://twitter.com/swizzard). 
```

Comment:
@hugovk Ah, do you think you'd have time to look at this?
```
 [swizzard](https://twitter.com/http://twitter.com/swizzard).
```

(If not, no worries, should be easy for me to fix later.)